scripts/build.sh

```diff
@@ -48,9 +50,14 @@ if "${do_build}"; then
         echo "Error: No .dockerignore file found in root; this is required so Docker caching works properly" >&2
         exit 1
     fi
-    docker build -t "${IMAGE_NAME}" -f "${root_dirpath}/Dockerfile" "${root_dirpath}"
-fi
-
-if "${do_publish}"; then
-    docker push "${IMAGE_NAME}"
+    push_flag=''
+    if "${do_publish}"; then
+        echo "Info: Publish flag set to true, will publish images to DockerHub"
+        push_flag='--push'
+    else
+        echo "Info: Publish flag set to false, nothing will be published to DockerHub"
+    fi
+    docker context create ${DOCKER_CONTEXT}
```

Comment:
@gbouv  suuuuper important that all variable references are surround by `""` per https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/master/bash-style-guide.md#bash-variables-should-always-be-wrapped-in-double-quotes-and-referenced-via--eg-echo-some_variable-unless-you-explicitly-want-word-splitting-very-rare !

Reply:
Thanks, I'll submit a new PR to fix that 🙏